packages/rule-components/src/ReportDetails/RuleFeedback.js

```diff
@@ -1,36 +1,58 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ThumbsUpIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-up-icon';
-import ThumbsDownIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-down-icon';
+import ThumbsUpIcon from '@patternfly/react-icons/dist/js/icons/thumbs-up-icon';
+import OutlinedThumbsUpIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-up-icon';
+import ThumbsDownIcon from '@patternfly/react-icons/dist/js/icons/thumbs-down-icon';
+import OutlinedThumbsDownIcon from '@patternfly/react-icons/dist/js/icons/outlined-thumbs-down-icon';
 import { Button } from '@patternfly/react-core';
 
 // ruleId - is id of current rule
 // onFeedbackSent(ruleId, vote) is a callback which is called when feedback is changed
 // where ruleId is id of the rule, vote is either -1, 0 or 1
-const RuleFeedback = ({ ruleId, onFeedbackChanged }) => (
-    <div>
-        <span>Is this helpful?</span>
-        <Button
-            className="ins-c-rule__rule-feedback-like-button"
-            variant="plain"
-            aria-label="Rule is helpful"
-            onClick={ () => onFeedbackChanged(ruleId, 1) }
-        >
-            <ThumbsUpIcon/>
-        </Button>
-        <Button
-            className="ins-c-rule__rule-feedback-dislike-button"
-            variant="plain"
-            aria-label="Rule is not helpful"
-            onClick={ () => onFeedbackChanged(ruleId, -1) }
-        >
-            <ThumbsDownIcon/>
-        </Button>
-    </div>
-);
+class RuleFeedback extends React.Component {
+    state = { feedbackSaved: false }
+    render() {
+        const { ruleId, userVote, onFeedbackChanged } = this.props;
+        return <div>
+            <span>Is this helpful?</span>
+            <Button
+                className="ins-c-rule__rule-feedback-like-button"
+                variant="plain"
+                aria-label="Rule is helpful"
+                onClick={ () => {
+                    this.setState({ feedbackSaved: true });
+                    onFeedbackChanged(ruleId, 1);
+                }}
+            >
+                {
+                    userVote === 1
+                        ? <ThumbsUpIcon color="#3C6C37"/>
```

Comment:
Please don't use direct colors, if we'll change color pallete in future this will be hard to fix. You should be good with `-pf-global--palette--green-500`

Reply:
I think we should ask @doruskova if she is fine with that color.

So you want to move color to CSS?

Reply:
@bond95 it's var(--pf-global--success-color--200)